platformio.ini

```diff
@@ -167,53 +175,48 @@ build_flags =
   -D RX_ELRS_GENERIC_2400_TD_PA_ESP32
 
 
-[env:rx-iflight-elrs-td-2400]
-extends = env:rx-generic-2400-td-pa
-build_flags =
-  ${env_common_rx_esp32.build_flags}
-  -D RX_ELRS_IFLIGHT_ELRS_TD_2400_ESP32
-
+;-- generic 2.4 GHz ELRS boards with overlays or other variations
 
-[env:rx-iflight-elrs-2400]
+[env:rx-iflight-elrs-2400]              ; like generic 2400 td pa, but provides 250 mW instead of 100 mW, has its own name
```

Comment:
Comment should read

'like generic 2400 pa, but provides 500 mW instead of 100 mW, has its own name'

Reply:
ups. THANKS! done.